samples/properties_user_links_audit_test.py

```diff
@@ -0,0 +1,25 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import properties_user_links_audit
+
+
+TEST_PROPERTY_ID = os.getenv("GA_TEST_PROPERTY_ID")
+
+
+def test_properties_user_links_audit(capsys):
+    properties_user_links_audit.audit_property_user_links(TEST_PROPERTY_ID)
+    out, _ = capsys.readouterr()
+    assert "Result" in out
```

Comment:
Are there any other assertions that can be made about the output?

Reply:
This test ensures that the request proto is valid and the call succeeds, which can be deducted by looking for "Result" string in the output. We do not currently have a test Google Analytics property that we can use as a stable snapshot for testing, but we are exploring this possibility.